pennylane/allocation.py

```diff
@@ -150,21 +150,17 @@ def deallocate(wires: DynamicWire | Wires | Sequence[DynamicWire]) -> Deallocate
 
     .. code-block:: python
 
-        @qml.qnode(qml.device('default.qubit'))
         def c():
             qml.H(0)
 
-            wires = qml.allocation.allocate(1, require_zeros=True)
+            wires = qml.allocate(1, require_zeros=True)
             qml.CNOT((0, wires[0]))
             qml.CNOT((0, wires[0]))
-            qml.allocation.deallocate(wires, restored=True)
+            qml.deallocate(wires, restored=True)
 
-            new_wires = qml.allocation.allocate(1)
+            new_wires = qml.allocate(1)
             qml.SWAP((0, new_wires[0]))
-            qml.allocation.deallocate(new_wires)
-
-            return qml.probs(wires=0)
-
+            qml.deallocate(new_wires)
```

Comment:
The drawing below shows `Probs`, but this qfunc doesn't return any measurements.

Reply:
```suggestion
            qml.allocation.deallocate(new_wires)
```